app/sidekiq/simple_forms_api/notification/send_notification_email_job.rb

```diff
@@ -11,11 +11,11 @@ class SendNotificationEmailJob
 
       attr_reader :notification_type, :config, :form_number, :user_account
 
-      def perform(notification_type:, form_submission_attempt:, user_account:)
+      def perform(notification_type:, form_submission_attempt:, form_number:, user_account:)
```

Comment:
I have to add this `form_number` argument because regular Simple Forms and uploaded forms handle the form number differently. It's awkward :/